OWNERS

```diff
@@ -1,22 +1,13 @@
 # Reviewers can /lgtm /approve but not sufficient for auto-merge without an
 # approver
 reviewers:
-- Rajakavitha1
-- stewart-yu
-- xiangpengzhao
-- zhangxiaoyu-zidif
+- makocchi-git
+- MasayaAoyama
 
 # Approvers have all the ability of reviewers but their /approve makes
 # auto-merge happen if a /lgtm exists, or vice versa, or they can do both
 # No need for approvers to also be listed as reviewers
 approvers:
-- bradamant3
-- bradtopol
-- chenopis
-- kbarnard10
-- mistyhacks
-- ryanmcginnis
-- steveperry-53
-- tengqm
-- zacharysarah
-- zparnold
+- cstoku
+- nasa9084
+- tnir
```

Comment:
Are these reviewers and approvers all members of Kubernetes organization? If not, I'm afraid they may not be able to lgtm/approve PRs in this repo.